src/main/webapp/about.jsp

```diff
@@ -74,6 +74,11 @@
                 <td><strong>Marcel Gonzalez:</strong>
                 Currently studying Computer Science at Tecnologico de Monterrey</td></tr>
 
+              <tr><td align="center"><img src="About-IMG/Default-Profile-IMG.png" class="prof-pic">
+                </td>
+                <td><strong>Aaron Colwell - PA:</strong>
+                Our super amazing project advisor who gave us so much support!</td></tr>
```

Comment:
Thanks for the kind words. I'll send a pull request with a profile image shortly. :)